test/faktory_worker/worker_test.exs

```diff
@@ -449,6 +449,39 @@ defmodule FaktoryWorker.WorkerTest do
       assert result.worker_state == :ok
     end
 
+    test "should log error, sleep, and schedule next fetch worker cannot connect" do
+      expect(FaktoryWorker.SocketMock, :connect, fn _host, _port, _opts ->
+        nil
+      end)
+
+      expect(FaktoryWorker.SocketMock, :connect, fn _host, _port, _opts ->
+        nil
+      end)
+
+      opts = [
+        worker_id: Random.worker_id(),
+        worker_module: TestQueueWorker,
+        retry_interval: 1,
+        connection: [socket_handler: FaktoryWorker.SocketMock]
+      ]
+
+      worker =
+        opts
+        |> Worker.new()
+
+      log_message =
+        capture_log(fn ->
+          Worker.send_fetch(worker)
+
+          consume_initial_fetch_message()
```

Comment:
Really picky but can we put this above the assertion to keep consistency please.